rules/avoid-injections.test.js

```diff
@@ -26,6 +26,13 @@ ruleTester.run("avoid-injections", rule, {
     "knex('users').whereRaw('id = ?', [1]);",
     "knex('users').whereRaw(`id = 1`);",
     "const joinCondition = `blog_posts ON users.id = blog_posts.author`; knex('users').select(['email']).joinRaw(joinCondition)",
+    `function thing1() {
+      return {
+        thing2: () => {},
+      };
+    }
+    thing1().thing2();
```

Comment:
This code is already passing, without the change. Can we add the snippet from the issue here instead?

Reply:
updated in https://github.com/AntonNiklasson/eslint-plugin-knex/pull/6/commits/add9101aec13ea040fd67548daeeff1c52e34910